FWCore/scripts/k4run

```diff
@@ -63,8 +64,8 @@ if __name__ == "__main__":
     # print a doc line showing the configured algorithms
     if run_with_options_file:
       for alg in ApplicationMgr().TopAlg:
-        print ' --> ', alg.name(), 
-      print "",'\n'
+        print(' --> ', alg.name(), )
```

Comment:
```suggestion
        print(' --> ', alg.name(), end='')
```
or does the comma at the end also prevent a new line with the print function?

Reply:
It does not, thanks!